R/calc_views.R

```diff
@@ -107,7 +107,9 @@ set_view_lim <- function(views){
   
 
   data <- list(y=summarize_args(views,c('y','y1','y0'),ignore=c('window','gs.config')), 
-               x=summarize_args(views,c('x','x1','x0'),ignore=c('window','gs.config')))
+               x=summarize_args(views,c('x','x1','x0'),ignore=c('window','gs.config')),
+               yaxs=summarize_args(views,c('yaxs'),ignore=c('window','gs.config')),
```

Comment:
I would call `x` and `y` data, but the `yaxs` and `xaxs` probably shouldn't be called that (just for readability's sake)